tests/e2e/default/contacts/edit-person-home-place.wdio-spec.js

```diff
@@ -0,0 +1,100 @@
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const usersAdminPage = require('@page-objects/default/users/user.wdio.page');
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const contactPage = require('@page-objects/default/contacts/contacts.wdio.page');
+const utils = require('@utils');
+
+describe('Edit Person Under Area', () => {
+  const placeFactory = require('@factories/cht/contacts/place');
+  const personFactory = require('@factories/cht/contacts/person');
+  const places = placeFactory.generateHierarchy();
+  const districtHospital = places.get('district_hospital');
+
+  const offlineUsername = 'jack_test';
+  const onlineUsername = 'program_officer';
+  const password = 'Jacktest@123';
+  const editedPlaceName = 'Updated Health Center';
+
+  const healthCenter = placeFactory.place().build({
+    name: 'HealthCenter',
+    type: 'health_center',
+    parent: {
+      _id: districtHospital._id,
+      parent: {
+        _id: '',
+      },
+    },
+  });
+
+  const offlinePerson = personFactory.build({
+    name: 'Jack',
+    parent: {
+      _id: healthCenter._id,
+      parent: healthCenter.parent,
+    },
+  });
+
+  const onlinePerson = personFactory.build({
+    name: 'Program Officer',
+    parent: {
+      _id: healthCenter._id,
+      parent: healthCenter.parent,
+    },
+  });
```

Comment:
You don't need to create a person and then manually create the user, you can create directly the user using the [userFactory](https://github.com/medic/cht-core/blob/7110-update-user-home-place/tests/factories/cht/users/users.js), like this:

[Offline user example:](https://github.com/medic/cht-core/blob/7110-update-user-home-place/tests/e2e/default/enketo/pregnancy.wdio-spec.js#L20-L29) 

```.js
const offlineUser = userFactory.build({ place: healthCenter._id, roles: ['chw'] });
await utils.createUsers([offlineUser]);
await loginPage.login(offlineUser);
```

[Online user example:](https://github.com/medic/cht-core/blob/7110-update-user-home-place/tests/e2e/default/contacts/fab-actionbar.wdio-spec.js#L11-L17)

```.js
const onlineUser = userFactory.build({ place: healthCenter._id, roles: [ 'program_officer' ] });
await utils.createUsers([ onlineUser ]);
await loginPage.login(onlineUser);
```